chain/messagepool/check_test.go

```diff
@@ -0,0 +1,224 @@
+//stm: #unit
```

Comment:
What's the meaning and purpose behind these stm comments?

Reply:
Those annotations are picked up by a custom test crawler that the Bloxico team implemented as a part of the [System Test Matrix](https://github.com/filecoin-project/system-test-matrix) dashboard. Here's what the [dashboard](https://lotus.systemtestmatrix.com/) looks like. 
It's something we've been working on for the previous couple of months, guided by @raulk and @BlocksOnAChain .
The plan is to keep all the tests annotated from now on, because it gives us a lot more context about the kinds of tests we have and the behaviors that are tested inside of those test than simple code coverage provides.

Reply:
Thanks I knew there'd be a story behind it!